drivers/claims_reporting/app/models/claims_reporting/quality_measures_report.rb

```diff
@@ -734,7 +734,7 @@ def medical_claim_based_rows
     private def in_set?(vs_name, claim, dx1_only: false)
```

Comment:
These changes seem to have snuck in again, I don't think that was intentional yes?